dev/com.ibm.websphere.appserver.features/visibility/public/jaspic-2.0/io.openliberty.jaspic-2.0.feature

```diff
@@ -0,0 +1,23 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jaspic-2.0
+visibility=public
+IBM-API-Package: \
+  jakarta.security.auth.message; type="spec", \
+  jakarta.security.auth.message.callback; type="spec", \
+  jakarta.security.auth.message.config; type="spec", \
+  jakarta.security.auth.message.module; type="spec"
```

Comment:
This feature depends on appSecurity-4.0 that already exports these packages.  Either it is redundant that they are exported here, or they should not be exported in appSecurity-4.0.  

In appSecurity-4.0 we do not export the IBM-SPI-Package.  So I believe we should either declare the SPIs in appSecurity-4.0 and remove the SPI package in here and remove the -jars and files or we should remove the -jar and files from appSecurity-4.0 and leave everything else as is.

Reply:
I am not sure why appSecurity-3.0 was exporting these packages. Would seem you would only do so when using the jaspic feature?

Honestly, I tried not to change much of the feature files more than I needed to (i.e. b/c a bundle or feature name changed). I agree, we should probably remove the -jar from appSecurity-4.0, since I believe the -jar is specific to jaspic.

Maybe there is some reason I am unaware of for why the appSecurity feature was exporting these packages and exposing the JAR.

Reply:
These are jaspic specific so I am not sure why they are exported in appSecurity-3.0 / 4.0. I don't know why someone using appSecurity-4.0, which does not depend on the jaspic-2.0 feature, would expect these artifacts and APIs to be there.